http-client/Network/HTTP/Client/Request.hs

```diff
@@ -122,9 +122,9 @@ throwErrorStatusCodes req res = do
 -- You can place the request method at the beginning of the URL separated by a
 -- space, e.g.:
 --
--- @@@
```

Comment:
There's no `@@@` syntax afaik. This leads to example rendering like this

![Screenshot from 2023-01-15 10-44-28](https://user-images.githubusercontent.com/2716069/212534031-18a5683f-09f6-4ad1-9ad9-f5e344e1b189.png)


After this fix:

![Screenshot from 2023-01-15 10-43-15](https://user-images.githubusercontent.com/2716069/212534062-501544d8-ec78-4d87-9039-84eb6b8548ab.png)